### PR TITLE
Rename level 2

### LINKED ISSUE #841: Rename and rearrange Level 2
Level 2 is entitled as "Basic Finesses". The subtitle reads "Special Moves" and the text starts with the "5 Stall"

Proposal:

- Rename Level 2 to "Basic Moves"
- Rename the subtitle to "Finesses and Stalls"
- Move "5 Stall" after the last finesse (The Self Finesse)

### PR DESCRIPTION
**Pull request by DrAnax**
Closes #841

### CODE REVIEW

File: docs/level-2.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ import EarlyGameQuestion4 from '@site/image-generator/yml/beginner/early-game-qu
 
 <br />
 
-## Special Moves
+## Stalls and Finesses
```
Comment (Zamiell) [inline docs/level-2.md:22]: special moves is a special header that every single level has, why rename this?

Comment (DrAnax) [reply docs/level-2.md:22]: It was in my original proposal and you didn't object.

Comment (Zamiell) [reply docs/level-2.md:22]: i didnt understand it properly I guess.
every single level should have header called "Special Moves" for uniformity.

Comment (DrAnax) [reply docs/level-2.md:22]: No problem at all, resending

**Comment by Zamiell** [comment]
ok thanks :)

**Pull request merged**